brands/tourism/hotel.json

```diff
@@ -625,6 +722,25 @@
       "tourism": "hotel"
     }
   },
+  "tourism/hotel|SpringHill Suites": {
+    "countryCodes": ["ca", "mx", "us"],
+    "tags": {
+      "brand": "SpringHill Suites",
+      "brand:wikidata": "Q7580351",
+      "brand:wikipedia": "en:SpringHill Suites",
+      "name": "SpringHill Suites",
+      "tourism": "hotel"
+    }
+  },
+  "tourism/hotel|St. Regis": {
```

Comment:
Add “Saint Regis” as a matching name and perhaps an `alt_name`. Or maybe make “St. Regis” the `short_name` and “Saint Regis” the `name`. (The rule against abbreviations isn’t as strongly observed with brand names, but it still seems apt for an ambiguous abbreviation like “St.”)

Reply:
Done